src/services/expenseService.js

```diff
@@ -0,0 +1,71 @@
+/* eslint-disable function-paren-newline */
```

Comment:
Can we fix this eslint issue?

Reply:
Seems like an eslint rules conflict...
<img width="780" alt="Screenshot 2024-05-16 at 13 14 23" src="https://github.com/mate-academy/node_accounting-app-v2/assets/139244901/116860e9-6c92-440f-b758-f48ec7cf4adc">
>>>